odyssey.conf

```diff
@@ -210,7 +210,7 @@ cache_coroutine 0
 #
 # 16KB by default.
 #
-coroutine_stack_size 4
+coroutine_stack_size 8
```

Comment:
Is this leftover?

Reply:
without this, coroutines do not have enough memory on the stack to call the pam library functions